test/cljs/replumb/repl_test.cljs

```diff
@@ -105,6 +106,38 @@
       (is (= "nil" dirstring) "(dir clojure.string) should return valid docstring")
       (repl/reset-env!)))
 
+  (deftest process-apropos
+    (let [res (read-eval-call "(apropos \"tim\")")
+          result (unwrap-result res)
+          expected "(cljs.core/-deref-with-timeout cljs.core/dotimes cljs.core/system-time cljs.core/time)"]
+      (is (success? res) "(apropos \"tim\") should succeed")
+      (is (valid-eval-result? result) "(apropos \"tim\") should be a valid result")
+      (is (= expected result) "(apropos \"tim\") should return valid docstring")
+      (repl/reset-env!))
+
+    (let [res (read-eval-call "(apropos #\"tim\")")
+          result (unwrap-result res)
+          expected "(cljs.core/-deref-with-timeout cljs.core/dotimes cljs.core/system-time cljs.core/time)"]
+      (is (success? res) "(apropos #\"tim\") should succeed")
+      (is (valid-eval-result? result) "(apropos #\"tim\") should be a valid result")
+      (is (= expected result) "(apropos #\"tim\") should return valid docstring")
+      (repl/reset-env!))
+
+    (let [res (read-eval-call "(apropos #\"t[i]me\")")
+          result (unwrap-result res)
+          expected "(cljs.core/-deref-with-timeout cljs.core/dotimes cljs.core/system-time cljs.core/time)"]
+      (is (success? res) "(apropos  #\"t[i]me\") should succeed")
+      (is (valid-eval-result? result) "(apropos  #\"t[i]me\") should be a valid result")
+      (is (= expected result) "(apropos #\"t[i]me\") should return valid docstring")
+      (repl/reset-env!))
+
+    (let [res (read-eval-call "(apropos \"t[i]me\")")
+          result (unwrap-result res)]
+      (is (success? res) "(apropos \"t[i]me\") should succeed")
+      (is (valid-eval-result? result) "(apropos  #\"t[i]me\") should be a valid result")
+      (is (= "nil" result) "(apropos \"t[i]me\") should return valid docstring")
```

Comment:
"should return \"nil\"" but what is the difference with the previous? I cannot see it, maybe I am tired :smile: